pkg/api/v20230401/openshiftcluster.go

```diff
@@ -126,13 +126,23 @@ type ServicePrincipalProfile struct {
 	ClientSecret string `json:"clientSecret,omitempty" mutable:"true"`
 }
 
+type OutboundType string
+
+const (
+	OutboundTypeUserDefinedRouting OutboundType = "UserDefinedRouting"
+	OutboundTypeLoadbalancer       OutboundType = "Loadbalancer"
+)
+
 // NetworkProfile represents a network profile.
 type NetworkProfile struct {
 	// The CIDR used for OpenShift/Kubernetes Pods.
 	PodCIDR string `json:"podCidr,omitempty"`
 
 	// The CIDR used for OpenShift/Kubernetes Services.
 	ServiceCIDR string `json:"serviceCidr,omitempty"`
+
+	// The OutboundType use for egress traffic.
+	OutboundType OutboundType `json:"outboundType,omitempty"`
```

Comment:
We still have `mutable:true` in the admin API so you can manually patch the clusters which we did this to from a support exception.  We don't need the `mutable:true` flag on this, just calling it out as something we need to get back to eventually.  

I'll put in a PR with a TODO as a follow-up item when we GA day 2 changing of outbound type.